.github/workflows/tests.yml

```diff
@@ -27,14 +24,11 @@ jobs:
   toolkit-tests:
     runs-on: ubuntu-latest
     needs: build
-    strategy:
-      matrix:
-        node-version: [16.x, 18.x]
```

Comment:
Why do we remove matrix ? 
We don't want to test on multiple node versions anymore ? 
I think it's nice to test on LTS versions, and so, Node 18 and 20 (https://endoflife.date/nodejs)